src/parser/grammar.jison

```diff
@@ -245,9 +245,9 @@ bhImportIdentifier
   |	Ident
     { $$ = { name: yy.ident($1, @1), alias: yy.ident($1, @1), isTypeof: false } }
   | TYPEOF Ident AS Ident
-    { $$ = { name: yy.ident($1, @1), alias: yy.ident($3, @3), isTypeof: true } }
+    { $$ = { name: yy.ident($2, @2), alias: yy.ident($4, @4), isTypeof: true } }
   | TYPEOF Ident
-    { $$ = { name: yy.ident($1, @1), alias: yy.ident($1, @1), isTypeof: true } }
```

Comment:
Hasn't this been fixed in previous commits?

Reply:
My thought exactly, but obviously not.